Extensions/shuffle+.js

```diff
@@ -335,13 +464,97 @@
             });
         }
 
-        success(count);
         Spicetify.Player.next();
+
+        switch (type) {
+            case Type.ARTIST:
+                if (CONFIG.artistMode == "topTen") {
+                    Spicetify.showNotification(`Shuffled Top ${count} Songs`);
+                    break;
+                }
+                if (CONFIG.artistMode == "likedSongArtist") {
+                    Spicetify.showNotification(`Shuffled ${count} Liked Songs`);
+                    break;
+                }
+                if (CONFIG.artistMode == "single") {
+                    Spicetify.showNotification(`Shuffled ${artistFetchTypeCount.single} Singles, Totally ${count} Songs`);
```

Comment:
```suggestion
                    Spicetify.showNotification(`Shuffled ${artistFetchTypeCount.single} Singles, Total of ${count} Songs`);
```
Same for the cases below.

Reply:
mo